packages/zap/zap.pacscript

```diff
@@ -0,0 +1,23 @@
+name="zap"
+version="2.2.1"
+depends="curl grep jq wget"
+url="https://github.com/srevinsaju/zap/releases/download/v2.2.1/zap-amd64"
```

Comment:
Find the hash, and include that as described in the wiki

Reply:
sha256sum hash has been recorded and added.